.circleci/config.yml

```diff
@@ -255,7 +255,7 @@ jobs:
           name: Deploy aws-uswest2.pangeo.io
           when: always
           command: |
-            hubploy deploy icesat2 pangeo-deploy ${CIRCLE_BRANCH} --force
+            hubploy deploy icesat2 pangeo-deploy ${CIRCLE_BRANCH} --force --atomic --cleanup-on-fail
```

Comment:
I dont think using both --atomic and --cleanup-on-fail will make sense, atomic is doing what cleanup does but a bit more than the cleanup flag as i understand it.

I suggest using only the atomic flag to have more comprehensible code down the line.